src/demo/index.php

```diff
@@ -115,7 +115,7 @@ function gtag() { dataLayer.push(arguments); }
     </div>
 
     <a href="javascript:toggleTheme()" class="darkmode" title="toggle dark mode">
-        <i class="<?php echo $_COOKIE["darkmode"] == "on" ? 'gg-sun' : "gg-moon"; ?>"></i>
+        <i class="<?php echo (isset($_COOKIE["darkmode"]) && !empty($_COOKIE) && $_COOKIE["darkmode"] == "on") ? 'gg-sun' : "gg-moon"; ?>"></i>
```

Comment:
isset should be enough, don't need both

```suggestion
        <i class="<?php echo (isset($_COOKIE["darkmode"]) && $_COOKIE["darkmode"] == "on") ? 'gg-sun' : "gg-moon"; ?>"></i>
```

Reply:
Done